packages/keycloak-user-management/src/components/Loading/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
I am completely against copying this component into this package!

Reply:
Likely going to be addressed via https://github.com/OpenSRP/web/pull/58